pkg/virt-operator/resource/generate/components/validatingadmissionpolicy.go

```diff
@@ -147,8 +148,21 @@ func NewHandlerV1ValidatingAdmissionPolicy(virtHandlerServiceAccount string) *ad
 					Name:       "newAnnotations",
 					Expression: "object.metadata.annotations",
 				},
+				{
+					Name:       "requestMatchNode",
+					Expression: "object.metadata.name == request.userInfo.extra['authentication.kubernetes.io/node-name'][0]",
+				},
+				{
+					Name:       "hasNode",
+					Expression: "('authentication.kubernetes.io/node-name' in request.userInfo.extra)",
+				},
 			},
 			Validations: []admissionregistrationv1.Validation{
+				{
+					Expression: "variables.hasNode && variables.requestMatchNode",
+					Message:    NodeRestrictionErrModifyAnother,
+					Reason:     pointer.P(metav1.StatusReasonForbidden),
```

Comment:
would this show the node name and the node in the request? Otherwise, it will be useless tbh

Reply:
No, let me see if message can take expression